### DISCUSSION TITLE
SwaggerHub link somewhere?

### DISCUSSION THREAD

**Discussion by rafaelmag110** [eclipse-edc/Connector#3950] [General]
I know Connector APIs are published in SwaggerHub every new release. However I can't find any link in this repository that points to the respective SwaggerHub release. Any reason for it not to be linked here?

**Comment by ndr-brt** [comment]
you can find them on the documentation website: https://eclipse-edc.github.io/docs/#/hands-on, currently only Connector has published specs

please be aware that we're moving all the documentation to github pages, the current snapshot version can be browsed here:
https://eclipse-edc.github.io/Connector/openapi/management-api/0.5.2-SNAPSHOT/

official announcement and link update will be done after next release

**Comment by arnoweiss** [comment]
There's something live here.

https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.7.1-SNAPSHOT#/Contract%20Definition%20V2/updateContractDefinitionV2

**Reply by ndr-brt** [reply]
please note that swaggerhub publication has been disbanded, now the api docs is published on github pages.
Links can be found in the menu under component -> api on the documentation site: https://eclipse-edc.github.io/docs
